components/content/inspira/ui/link-preview/LinkPreview.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <div class="relative inline-block">
```

Comment:
What if user wants to update the class of this `div` elements? Can you make the main class prop attach to this, and for NuxtLink, you can add another class prop like `linkClass`